utils/utils.go

```diff
@@ -138,6 +138,58 @@ func HandlePanic() {
 	}
 }
 
+// GenerateGithubActions generates a GitHub Actions workflow file for Keploy
+func GenerateGithubActions(logger *zap.Logger, path string, appCmd string) {
+	// Determine the path based on the alias "keploy"
+	var keployPath string
+	isDockerCmd := len(os.Getenv("IS_DOCKER_CMD")) > 0
+	if isDockerCmd {
+		logger.Info("Running in docker environment, skipping the path determination")
+		keployPath = "./"
+	} else {
+		logger.Info("Determining the path of the keploy binary based on the alias \"keploy\"")
+		keployPath = "/usr/local/bin/keploybin" // Default path
+		aliasCmd := exec.Command("which", "keploy")
+		aliasOutput, err := aliasCmd.Output()
+		if err == nil && len(aliasOutput) > 0 {
+			keployPath = strings.TrimSpace(string(aliasOutput))
+		}
+		logger.Info("Path of the keploy binary determined successfully", zap.String("path", keployPath))
+	}
+	// Define the content of the GitHub Actions workflow file
+	actionsFileContent := `name: Keploy
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, reopened, synchronize]
+jobs:
+  e2e-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Test-Report
+        uses: keploy/testgpt@main
```

Comment:
why exactly are we using testgpt?

Reply:
I assumed the way we are preparing this github actions workflow was in conjuction with TestGPT, if thats not the case then we will need to do a full keploy installation and prepare the workflow. I will get started on implementing that.